lib/connect.js

```diff
@@ -460,8 +467,11 @@ function connect(token, options) {
  *   downlink video bandwidth in bits per second (bps). By default, there are no limits on
  *   the downlink video bandwidth.
  * @property {boolean} [idleTrackSwitchOff] - Optional parameter that when enabled switches off a
- *    RemoteVideoTrack when no video element is attached to the track, or when all attached video
+ *    {@link RemoteVideoTrack} when no video element is attached to the track, or when all attached video
  *    elements of the track are not visible, or when the Document is not visible. This is enabled by default
```

Comment:
Please mention that this is applicable to Group and Small Group Rooms only.

Reply:
These two are properties of VideoBandwidthProfileOptions, which itself is group room specific.  We do specify that  `BandwidthProfileOptions` are group room specific on line 447